sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/PluginReferentialManifestConverter.java

```diff
@@ -36,7 +36,9 @@ public static PluginReferential fromPluginManifests(List<PluginManifest> pluginM
       plugin.merge(pluginManifest);
 
       Release release = new Release(plugin, pluginManifest.getVersion());
-      release.addRequiredSonarVersions(Version.create(pluginManifest.getSonarVersion()));
+      release.addRequiredSonarVersions(Product.OLD_SONARQUBE, Version.create(pluginManifest.getSonarVersion()));
```

Comment:
I am not understanding how the same value from the pluginManifest can be used with the three different products.

Reply:
We only get one value from plugin manifest. I haven't heard about any plans to modify plugins manifests, so so far I don't have any other value to work with. This pull request is about marketplace and HTMLS/JSONs.

Reply:
I am double checking it again.

Reply:
So this is actually plugin-api version. I think the code is wrong here to call `addRequiredSonarVersions`. I think we should drop this field all together as it not used in the code anymore.
See the method org.sonar.server.plugins.PluginReferentialMetadataConverter#toPluginManifest.

For this PR I will leave it out of scope.